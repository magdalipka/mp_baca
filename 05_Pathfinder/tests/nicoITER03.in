10 10
1 0 1 0 0 1 0 1 0 0
0 0 1 0 1 0 0 0 1 0
0 1 0 0 1 0 1 0 0 0
0 0 1 0 0 0 1 0 1 1
1 0 0 0 1 0 1 0 1 0
0 0 1 1 0 0 1 0 1 0
0 1 0 0 0 1 1 0 0 0
1 1 0 1 1 1 0 0 1 0
0 0 0 1 0 0 0 1 1 0
0 1 1 1 1 1 1 0 0 0
1
i 0 3 0 0

