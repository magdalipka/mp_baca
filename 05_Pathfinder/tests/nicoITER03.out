i N E N E E N E E S S W S W W S S W W S
